index.js

```diff
@@ -9,13 +10,18 @@ var email = argObject.email || argObject.e || argList[0];
 var message = argObject.message || argObject.m || argList.slice(1).join(" ");
 var isAnonymous = argObject.anonymous ? 1 : 0 || argObject.a ? 1 : 0;
 
+var appDirectory = process.env.HOME + "/.cheers";
+var dataPath = appDirectory + "/data.json";
```

Comment:
@conorhastings can you remove these two lines and ping me? They should have been moved to token js but I didn't do it until after you forked :(